### DISCUSSION TITLE
I want to request a feature about alias

### DISCUSSION THREAD

**Discussion by JenuelDev** [jenssegers/laravel-mongodb#2156] [Q&A]
So for example in SQL we have the `select user_id as id from users`
using as so that it will return data with a property name id instead of user_id.
sample output:

```php
$users = [
     {
          id: 1
      },
     {
          id: 2
      },
     {
          id: 3
      },
     {
          id: 4
      },
     {
          id: 5
      },
]
```

How can this be done in lumen MongoDB? thanks 👍🏻

**Comment by divine** [comment]
Hello,

Please stop spamming new issues, you're asking questions and that's the reason why they should go to discussions.

Thanks!

**Reply by JenuelDev** [reply]
oh no sorry, I'm not trying to spam, I'm just trying to at least request a new feature because I am using this library and In the docs.
Im just trying to request a feature for alias.

like for example, I like to have a query something like this. So that it will be easy to automatically alias a column.
using AS.
```
$users =
    User::where('user_id', '123')
        })->get(['firstName AS name']);

return json_encode($users)
```

So instead of returning this
```
[
  {
    firstName: 'Mark'
  },
  {
    firstName: 'John'
  },
]
```
It will return something like this
```
[
  {
    name: 'Mark'
  },
  {
    name: 'John'
  },
]
```
